src/main/java/com/teradata/tpcds/Session.java

```diff
@@ -47,17 +52,26 @@ public Session(double scale, String targetDirectory, String suffix, Optional<Tab
 
     public Session(double scale, String targetDirectory, String suffix, Optional<Table> table, String nullString, char separator, boolean doNotTerminate, boolean noSexism, int parallelism, int chunkNumber, boolean overwrite)
     {
+        this(scale, targetDirectory, suffix, table, nullString, separator, doNotTerminate, noSexism, parallelism,
```

Comment:
if the function arguments don't fit nicely on one line, can you put one per line?
```
this(
    scale,
    targetDirectory,
    ....)
```